src/game.cpp

```diff
@@ -1583,6 +1583,10 @@ bool game::do_turn()
     // consider a stripped down cache just for monsters.
     m.build_map_cache( levz, true );
     monmove();
+    double snow_level = get_weather().snow_level;
+    if( calendar::once_every( 1_minutes ) ) {
+        m.actualize_snow( snow_level );
```

Comment:
I suggest move it into the branch since it is not used outside
```suggestion
    if( calendar::once_every( 1_minutes ) ) {
        m.actualize_snow( get_weather().snow_level );
```